Engines/Wine/Shortcuts/Reader/script.js

```diff
@@ -1,122 +1,123 @@
 include("engines.wine.engine.object");
 
-var _WineShortcutReader = function (shortcut) {
-    var that = this;
-    that._shortcutManager = Bean("shortcutManager");
-    that._libraryManager = Bean("libraryManager");
-    that._uiQuestionFactory = Bean("uiQuestionFactory");
-    that._winePrefixesDirectory = Bean("propertyReader").getProperty("application.user.containers") + "/" + WINE_PREFIX_DIR + "/";
+class WineShortcutReader {
+    constructor(shortcut) {
+        this.shortcut = shortcut;
```

Comment:
We mainly access `this.shortcut` using `JSON.parse(this.shortcut.script)` maybe it makes sense to provide this value as either a getter method:

```javascript
get shortcutScript() {
   return JSON.parse(this.shortcut.script)
}
```

or as a public field.

What do you think @plata @ImperatorS79?

Reply:
NO IDEA ^^.

Reply:
I don't care too much about this detail but you're approach is fine for me.